lib/Alchemy/Phrasea/SearchEngine/Elastic/Indexer/Record/Hydrator/ThesaurusHydrator.php

```diff
@@ -22,12 +22,13 @@
 class ThesaurusHydrator implements HydratorInterface
 {
     private $thesaurus;
+    private $candidate_terms;
     private $helper;
 
-    public function __construct(Thesaurus $thesaurus, CandidateTerms $candidateTerms, RecordHelper $helper)
+    public function __construct(Thesaurus $thesaurus, CandidateTerms $candidate_terms, RecordHelper $helper)
```

Comment:
Right now, `Alchemy\Phrasea\SearchEngine\Elastic` is mainly in snake case. I prefer to keep this that way unless we change it everywhere.